### ISSUE TITLE
Panic in `insert_many` with many nullable fields when seeding data

### ISSUE THREAD

**Issue by CandleCandle** [loco-rs/loco#796]
**Description**

I think this is more a problem with [seaorm](https://github.com/SeaQL/sea-orm) than Loco, however, posting here gives the work-around exposure in the place where it is relevant. It may be sensible to implement a work-around in Loco.

https://github.com/SeaQL/sea-orm/issues/1407

**To Reproduce**

I have a table with several nullable columns. Loco takes the single yaml file as input and converts it to an `insert_many` call as part of the `seed`.

Observe that the entry at ID 2 does not set a value for `nullable1`, nor does the first entry set a value for `nullable2`.

Sample failing yaml:
```yaml
- id: 1
  something: some value
  nullable1: value
- id: 2
  something: some value
  nullable2: value
```

**Expected Behavior**

The above database entries to be created with missing fields being `null`.

**Environment:**

**Additional Context**

**Work-Arounds**

There are two possible work-arounds:
1. Each yaml file to include empty values for unused fields:
```yaml
- id: 1
  something: some value
  nullable1: value
  nullable2:
- id: 2
  something: some value
  nullable1:
  nullable2: value
```

2. group your inserts into separate files, one file per distinct set of nullable fields with non-null values. This separates it into many calls to `insert_many`

File 1:
```yaml
- id: 1
  something: some value
  nullable1: value
```

File 2:
```yaml
- id: 2
  something: some value
  nullable2: value
```

**Comment by jondot** [comment]
Thanks for the report, nice catch!
We'll think of a solution for this

**Comment by jondot** [comment]
Looks like this only mean we need to work around and insert one by one

**Comment by tyt2y3** [comment]
Thank you everyone. This has been resolved in SeaORM 1.1.3 https://github.com/SeaQL/sea-orm/issues/1407
